pmm/v3/pmm3-submodules.groovy

```diff
@@ -27,29 +27,42 @@ pipeline {
         label 'agent-amd64'
     }
     parameters {
-        choice(
-            choices: ['el9', 'next'],
-            description: 'The base OS of the image',
-            name: 'BASE_OS'
-        )
+        string(
+            defaultValue: 'v3',
+            description: 'Tag/Branch for pmm-submodules repository',
+            name: 'PMM_BRANCH')
+        string(
+            defaultValue: '',
+            description: 'URL for pmm-submodules repository PR',
+            name: 'CHANGE_URL')
+        string(
+            defaultValue: '',
+            description: 'ID for pmm-submodules repository PR',
+            name: 'CHANGE_ID')
+        string(
+            defaultValue: 'PR-2345',
```

Comment:
```suggestion
            defaultValue: '',
```
I guess it's a test value, right?

Reply:
@ademidoff It's a random value I've added just to ensure the users of the script about the format of the "BRANCH_NAME" variable (starts with PR-). Should I remove the value?

Reply:
This default value is only good for just one PR, so I'd remove it. We could leave a comment instead.

Reply:
Removed the default value and added a comment.